config/config-example.js

```diff
@@ -210,6 +210,17 @@ exports.appealurl = '';
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// Github plugin configuration
```

Comment:
This comment still doesn't meet the form of the rest of the Config comments.  Example of how this would fit:

```
// github - this is used for update alerts for a GitHub repository.
// To use this, uncomment this and fill in the object with the
// appropriate attributes.
/**exports.github = {
 	options: {
 		port: 3420, // the port for your webhook
 		secret: '', // the secret specified during webhook creation
 		rooms: [], // list of rooms to be notified on GitHub updates.  If left unspecified, it will check for a Development room.
};**/
```

Then, you could just not define it instead of leaving it false and having a commented out version.  I think that's probably the most uniform with the rest of the Config documentation.